unime/src/routes/credentials/[id]/DataUrlImageRenderer.svelte

```diff
@@ -0,0 +1,13 @@
+<script lang="ts">
+  export let key: string;
+  export let dataUrl: string;
+</script>
+
+<!--
+@component
+This renderer for `credentialSubject` fields renders an embedded image for a given [Data URL](https://developer.mozilla.org/en-US/docs/Web/URI/Schemes/data).
+-->
+<div class="flex flex-col items-center rounded-xl bg-background-alt px-4 py-3 text-[13px]/[24px]">
```

Comment:
Maybe rather not use `text-[13px]/[24px]` and switch to a default Tailwind text size? Would need to be done on the entire `/credentials` route.

Reply:
True, but I think redefining typography across the entire app should be a separate issue. We can also think about defining a custom class for `text-[13px]/[24px]` since it has 67 occurrences in the entire project.